src/on.ts

```diff
@@ -0,0 +1,97 @@
+import { Handle, EventObject } from './interfaces';
+import { createHandle, createCompositeHandle } from './lang';
+import Evented from './Evented';
+
+// Only used for Evented and EventEmitter, as EventTarget uses EventListener
+type EventCallback = (event: {}) => void;
```

Comment:
This also needs to be exported. Without this exported, dts-generator fails to generate a definition for the `on` module. You can check this with `grunt dist`.